test/contentsharecontroller/ContentShareMediaStreamBroker.test.ts

```diff
@@ -96,43 +115,64 @@ describe('ContentShareMediaStreamBroker', () => {
       const mediaDevices = navigator.mediaDevices;
       // @ts-ignore
       const getDisplayMediaSpy = sinon.spy(mediaDevices, 'getDisplayMedia');
-      const streamConstrait: MediaStreamConstraints = { video: true };
-      await contentShareMediaStreamBroker.acquireDisplayInputStream(streamConstrait);
-      getDisplayMediaSpy.calledOnceWith(streamConstrait);
+      await contentShareMediaStreamBroker.acquireDisplayInputStream(defaultStreamConstraints);
+      expect(getDisplayMediaSpy.calledWith(sinon.match(defaultStreamConstraints))).to.be.true;
     });
 
     it('getUserMedia if electron', async () => {
       // @ts-ignore
       const mediaDevices = navigator.mediaDevices;
       // @ts-ignore
       const getUserMedia = sinon.spy(mediaDevices, 'getUserMedia');
-      const streamConstrait: MediaStreamConstraints = {
-        audio: false,
-        video: {
-          // @ts-ignore
-          mandatory: {
-            chromeMediaSource: 'desktop',
-            chromeMediaSourceId: 'sourceId',
-            maxFrameRate: 3,
-          },
-        },
-      };
-      await contentShareMediaStreamBroker.acquireDisplayInputStream(streamConstrait);
-      getUserMedia.calledOnceWith(streamConstrait);
+      await contentShareMediaStreamBroker.acquireDisplayInputStream(
+        defaultElectronStreamConstraints
+      );
+      expect(getUserMedia.calledWith(sinon.match(defaultElectronStreamConstraints))).to.be.true;
     });
   });
 
   describe('acquireScreenCaptureDisplayInputStream', () => {
     it('with source Id', async () => {
       const spy = sinon.spy(contentShareMediaStreamBroker, 'acquireDisplayInputStream');
       await contentShareMediaStreamBroker.acquireScreenCaptureDisplayInputStream('sourceId');
-      spy.calledOnceWith(sinon.match.any);
+      expect(spy.calledWith(sinon.match(defaultElectronStreamConstraints))).to.be.true;
     });
 
     it('without source Id', async () => {
       const spy = sinon.spy(contentShareMediaStreamBroker, 'acquireDisplayInputStream');
-      await contentShareMediaStreamBroker.acquireScreenCaptureDisplayInputStream('sourceId');
-      spy.calledOnceWith(sinon.match.any);
+      await contentShareMediaStreamBroker.acquireScreenCaptureDisplayInputStream();
+      expect(spy.calledWith(sinon.match(defaultStreamConstraints))).to.be.true;
+    });
+
+    it('with non-default framerate', async () => {
+      const streamConstraints: MediaStreamConstraints = {
+        audio: false,
+        video: {
+          frameRate: {
+            max: 30,
+          },
+        },
+      };
+      const spy = sinon.spy(contentShareMediaStreamBroker, 'acquireDisplayInputStream');
+      await contentShareMediaStreamBroker.acquireScreenCaptureDisplayInputStream(null, 30);
```

Comment:
```
async acquireScreenCaptureDisplayInputStream(sourceId?: string, frameRate?: number)
```

The type of `sourceId` is implicitly `string | undefined` but not `null`. You can pass `undefined` instead.

Reply:
Andrew suggested the `string | null` type for `startContentShareFromScreenCapture()`. You can update that signature instead of changing the test file.